DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -1,10 +1,20 @@
-﻿@using DigitalLearningSolutions.Web.Helpers
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.Extensions
+@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.Models.Enums
 @using Microsoft.Extensions.Configuration
 @inject IConfiguration Configuration
 
 @{
   var headerExtension = (string)ViewData["Application"];
   var shouldDisplayHelpMenuItem = headerExtension != "Tracking System";
+  var selectedTab = (Tab)ViewData["SelectedTab"];
+  var applicationType = (ApplicationType)ViewData["ApplicationType"];
+  if (applicationType != null) {
+    ViewData["Application"] = applicationType.HeaderExtension;
+    ViewData["HeaderPath"] = applicationType.HeaderPath;
+    ViewData["HeaderPathName"] = applicationType.HeaderPathName;

```

Comment:
For now I'd rather not overwrite these even if applicationType is set. Doing so leaves a small possibility we'll change something on a page that we didn't mean to (because ViewData["ApplicationType"] can easily be set outside of the attribute we've made).
Can you just add in a null coalesce? `ViewData["Application"] = ViewData["Application"] ?? applicationType.HeaderExtension;`

Reply:
Cool, I've changed it to use compound assignment